scripts/parser.js

```diff
@@ -63,9 +63,38 @@ async function getJobsinDay(){
     await mbo.class.classes({
         "StartDateTime" : presentDateString,
         "EndDateTime" : tomorrowDateString
-    }, processResponse);
+    }, processClasses);
+}
+
+/**
+ * Using the class retrieved by API call, adds emails of 
+ * attendees into an array.
+ */
+async function processEnrollments(err, data){
+    if (err){
+        console.error("Failed to retrieve enrollments for the class", err);
+    } else {
+        const clients = data.Classes[0].Clients;
+        const emails = [];
```

Comment:
you don't have to use the push function, you can use the Promise.all async function to add into the array:
Syntax: 
```
const emails = await Promise.all(arr.map(async function(element) {
  return element;
}));
```